docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,69 +3,75 @@ title: Database Access AWS IAM Reference
+print-iam`](../../database-access/reference/cli.mdx#teleport-db-configure--awsprint-iam)
 command. For example, the following command would generate and print the IAM
 policies:
 
 ```code
-$ teleport db configure bootstrap --manual
+$ teleport db configure aws print-iam --types rds,redshift --role db-service-role
 ```
 
-Or if you prefer, you can manage the IAM permissions yourself. Examples of
-policies for each discovery type are shown below.
+To learn more about a specific database, please refer to the related section
+below.
 
-### Aurora/RDS
+## RDS
 
-<Tabs>
-  <TabItem label="RDS / IAM user">
-  Use this policy if you're connecting to RDS instances and your Teleport
-  Database Service instance runs as an IAM user (for example, uses an AWS
-  credentials file).
+(!docs/pages/includes/database-access/reference/aws-db-brief.mdx db="RDS instances, Aurora clusters, and RDS Proxies" !)
 
-  Replace `{account-id}` with your AWS Account ID:
-  ```json
-  {
-      "Version": "2012-10-17",
-      "Statement": [
-          {
-              "Effect": "Allow",
-              "Action": [
-                  "rds:DescribeDBInstances",
-                  "rds:ModifyDBInstance"
-              ],
-              "Resource": "*"
-          },
-          {
-              "Effect": "Allow",
-              "Action": [
-                  "iam:GetUserPolicy",
-                  "iam:PutUserPolicy",
-                  "iam:DeleteUserPolicy"
-              ],
-              "Resource": "arn:aws:iam::{account-id}:user/sample-user"
-          }
-      ]
-  }
-  ```
-  </TabItem>
-  <TabItem label="RDS / IAM role">
-  Use this policy if you're connecting to RDS instances and your Teleport
-  Database Service runs as an IAM role (for example, on an EC2 instance with
-  an attached IAM role).
+### RDS Auto-discovery
+
+The following IAM permissions are required for the Teleport service that
+performs auto-discovery:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "rds:DescribeDBInstances",
+                "rds:DescribeDBClusters",
+                "rds:DescribeDBProxies",
+                "rds:DescribeDBProxyEndpoints",
+                "rds:ListTagsForResource",
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+(!docs/pages/includes/database-access/reference/rds-action-notes.mdx!)
 
-  Replace `{account-id}` with your AWS Account ID:
+(!docs/pages/includes/database-access/reference/boundary-policy-note.mdx!)
+
+### RDS Access
+
+Teleport Database Service can automatically manage IAM policies of the attached
+IAM identity for RDS access. To use this feature, Teleport requires
+`iam:PutRolePolicy` or `iam:PutUserPolicy` permissions to grant itself
+necessary IAM permissions for each registered database.
+
+If you prefer to manage IAM identities manually, please refer to the [manage
```

Comment:
I think we can omit "please", since docs readers should expect terse imperative statements.